tests/Fixer/ClassNotation/NoUnneededFinalMethodFixerTest.php

```diff
@@ -211,6 +211,9 @@ class Foo {
             'abstract-class' => [
                 '<php abstract class Foo { final public function foo() {} }',
             ],
+            'abstract-construct' => [
+                '<?php abstract class Foo { private static $var; final public function foo() {} }',
```

Comment:
This actually affects all these cases where we have a `final` method that follows a `private` property - shall we rename the case, then?

Reply:
Done